CleverTap/Runtime/Native/UnityNativeWrapper/EventBuilders/UnityNativeBaseEventBuilder.cs

```diff
@@ -9,11 +9,13 @@ internal class UnityNativeEventBuilder {
         private UnityNativeDeviceInfo _deviceInfo;
         private UnityNativeSessionManager _sessionManager;
         private UnityNativeNetworkEngine _networkEngine;
-
-        internal UnityNativeEventBuilder(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine) {
+        private UnityNativeEventValidator _eventValidator;
+        
+        internal UnityNativeEventBuilder(UnityNativeCoreState coreState, UnityNativeNetworkEngine networkEngine, UnityNativeEventValidator eventValidator) {
```

Comment:
eventValidator is unused, so it is not needed

Reply:
Removed